src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -473,13 +502,30 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 			if let hoursSinceTestRegistration = storedUsageData.hoursSinceTestRegistration {
 				$0.hoursSinceTestRegistration = hoursSinceTestRegistration
 			}
-			if let daysSinceMostRecentDateAtRiskLevelAtTestRegistration = storedUsageData.daysSinceMostRecentDateAtRiskLevelAtTestRegistration {
-				$0.daysSinceMostRecentDateAtRiskLevelAtTestRegistration = daysSinceMostRecentDateAtRiskLevelAtTestRegistration
+			if let daysSinceMostRecentDateAtENFRiskLevelAtTestRegistration = storedUsageData.daysSinceMostRecentDateAtENFRiskLevelAtTestRegistration {
+				$0.daysSinceMostRecentDateAtRiskLevelAtTestRegistration = daysSinceMostRecentDateAtENFRiskLevelAtTestRegistration
 			}
-			if let hoursSinceHighRiskWarningAtTestRegistration = storedUsageData.hoursSinceHighRiskWarningAtTestRegistration {
-				$0.hoursSinceHighRiskWarningAtTestRegistration = hoursSinceHighRiskWarningAtTestRegistration
+			if let hoursSinceENFHighRiskWarningAtTestRegistration = storedUsageData.hoursSinceENFHighRiskWarningAtTestRegistration {
+				$0.hoursSinceHighRiskWarningAtTestRegistration = hoursSinceENFHighRiskWarningAtTestRegistration
+			}
+			
+			if let daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration = storedUsageData.daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration {
+				$0.ptDaysSinceMostRecentDateAtRiskLevelAtTestRegistration = daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration
+			}
+			if let hoursSinceCheckinHighRiskWarningAtTestRegistration = storedUsageData.hoursSinceCheckinHighRiskWarningAtTestRegistration {
+				$0.ptHoursSinceHighRiskWarningAtTestRegistration = hoursSinceCheckinHighRiskWarningAtTestRegistration
+			}
+			
+			// special handling for the triStateBool
+			if let submittedWithCheckins = storedUsageData.submittedWithCheckIns {
+				if submittedWithCheckins == true {
+					$0.submittedWithCheckIns = .tsbTrue
+				} else {
+					$0.submittedWithCheckIns = .tsbFalse
+				}
```

Comment:
$0.submittedWithCheckIns = submittedWithCheckins ? .tsbTrue : .tsbFalse

Reply:
Of course! 🤦‍♂️